uvloop/sslproto.pyx

```diff
@@ -301,6 +301,11 @@ cdef class SSLProtocol:
                 self._waiter.set_result(None)
         self._waiter = None
 
+    def _get_app_transport(self):
+        if self._app_transport is None:
+            self._app_transport = _SSLProtocolTransport(self._loop, self)
```

Comment:
We set `self._app_transport = None` in `connection_lost`.  I'm a bit worried that we might end up in a situation (accidentally or while debugging/refactoring) where we create `_SSLProtocolTransport` more than once for one protocol.  Thoughts?

Reply:
Yeah I had the same concern. How about adding an additional flag to make sure it's created only once?

Reply:
Yeah, feel free to add one.